python/neuroglancer/viewer_state.py

```diff
@@ -164,6 +164,12 @@ class OpacityTool(Tool):
     TOOL_TYPE = "opacity"
 
 
+@export_tool
+class VolumeRenderingScaleTool(Tool):
```

Comment:
Since this specifies number of depth samples (which is essentially 1/scale) a different name might be better, like volumeRenderingDepthSamples or similar.  Or we could change how it is represented in the JSON state, so that it actually is a "scale" parameter, but that adds other complications since the real scale depends on other things like the view frustum depth.

Reply:
I think changing the name is a better option, especially with the complications around the view frustum depth as you mention - thanks Jeremy!